Formula/pytorch.rb

```diff
@@ -0,0 +1,100 @@
+class Pytorch < Formula
+  desc "Tensors and dynamic neural networks"
+  homepage "https://pytorch.org/"
+  url "https://github.com/pytorch/pytorch.git",
+      tag:      "v1.12.1",
+      revision: "664058fa83f1d8eede5d66418abff6e20bd76ca8"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "eigen"
+  depends_on "libuv"
+  depends_on "numpy"
+  depends_on "openblas"
+  depends_on "openssl@1.1"
+  depends_on "protobuf"
+  depends_on "pybind11"
+  depends_on "python-typing-extensions"
+  depends_on "pyyaml"
+
+  on_macos do
+    depends_on arch: :arm64
```

Comment:
I'm fine with the rename, but I'm not fine with making this depend on ARM.

If it fails to build on Intel, then we need to understand why and fix it. Or, at least, report the problem upstream so that it can be fixed there.

Reply:
I just reported the problem upstream: https://github.com/pytorch/pytorch/issues/85956

Reply:
Thanks!